sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -286,11 +287,56 @@ def test_predict_pred_interval(Forecaster, fh, alpha):
             with pytest.raises(NotImplementedError, match="prediction intervals"):
                 f.predict(return_pred_int=True, alpha=alpha)
 
+def _check_predict_quantiles(pred_quantiles: list, y_train: pd.Series, fh, alpha):
+    # alpha can be a float or a list
+    # if a list, it has to be sorted
+    # check if alpha is increasing and in the given interval
+    if isinstance(alpha, list):
+        # check that alpha contains strictly increasing values
+        assert all(x < y for x, y in zip(alpha, alpha[1:]))
+        # check that alpha vales are in the [0, 1] interval
+        assert min(alpha) >= 0
+        assert max(alpha) <= 1
+    else:
+        assert 0 <= alpha <= 1
+
+    # check if the input is a dataframe
+    assert isinstance(pred_quantiles, pd.DataFrame)
+
+    # check time index (also checks forecasting horizon is more than one element)
+    _assert_correct_pred_time_index(pred_quantiles.index.levels[0], y_train.index[-1], fh)
+
+    # multiply variables with all alpha values
+    expected = pd.MultiIndex.from_product(y_train.columns, alpha)
+    assert all(expected == pred_quantiles.columns.to_flat_index())
+
+    # check if values are monotonically increasing
+    for var in pred_quantiles.columns.levels[0]:
+        for index in range(len(pred_quantiles.index)):
+            assert pred_quantiles[var].iloc[index].is_monotonic_increasing
```

Comment:
this doesn't always give the expected result, since the user could pass `alpha` in the list in a non-monotonic order? An easy way around this is to retrieve the alpha, and sort the columns before checking.